src/components/main/ItemCard.tsx

```diff
@@ -85,17 +85,19 @@ const ItemComponent = ({
   // because the following components are shared between the Grid and Table views
   const Actions = (
     <>
-      {member && member.id && <FavoriteButton size="medium" item={item} />}
       {enableEdition && (
+        <EditButton
+          item={
+            // DO NOT REMOVE cast
+            // here we cast explicitly to be equivalent to the grid which does not let us use Records
+            item.toJS() as DiscriminatedItem
+          }
+        />
+      )}
+      {member && member.id && (
         <>
-          <EditButton
-            item={
-              // DO NOT REMOVE cast
-              // here we cast explicitly to be equivalent to the grid which does not let us use Records
-              item.toJS() as DiscriminatedItem
-            }
-          />
           <DownloadButton id={item.id} name={item.name} />
```

Comment:
Actually, the Download button should be also available if the item is public, eg the member is not defined. You can get `isPublic` from `itemStatuses`.